IOPool/Common/bin/CollUtil.cc

```diff
@@ -97,6 +98,102 @@ namespace edm {
     }
   }
 
+  namespace {
+    class BranchBasketBytes {
+    public:
+      BranchBasketBytes(TBranch const *branch)
+          : basketFirstEntry_(branch->GetBasketEntry()),
+            basketBytes_(branch->GetBasketBytes()),
+            branchName_(branch->GetName()),
+            maxBaskets_(branch->GetMaxBaskets()) {}
+
+      std::tuple<Long64_t, unsigned> bytesInNextCluster(Long64_t clusterEnd) {
```

Comment:
How about a comment on what the two items in the tuple are?

Reply:
Added a comment